cmd/kubeadm/app/constants/constants.go

```diff
@@ -191,7 +191,7 @@ const (
 	// Default behaviour is 24 hours
 	DefaultTokenDuration = 24 * time.Hour
 
-	// LabelNodeRoleMaster specifies that a node is a master
+	// LabelNodeRoleMaster specifies that a node is a control-plane
 	// This is a duplicate definition of the constant in pkg/controller/service/service_controller.go
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
this is an exception keep it like `LabelNodeRoleMaster`.

Reply:
@vanduc95 
should have deleted the above comment next to deleting the similar one.

also i'm not sure whether we should keep this or change to `LabelNodeRoleMaster` at this point.
mostly leaning towards change.

but let's wait for more comments...

@rosti @kubernetes/sig-cluster-lifecycle-pr-reviews

Reply:
We need to keep it like `LabelNodeRoleMaster`. There is currently a constant named in the same way and used for the same purpose in `pkg/controller/service/service_controller.go` and the idea is at some point to move that constant to a common place (where we won't import a huge package just for it).
Until then, it's best to keep the constant names in sync.

Reply:
ok, keeping this like so.